support-frontend/assets/helpers/redux/checkout/checkoutSelectors.ts

```diff
@@ -42,35 +31,33 @@ export function getAllErrorsForContributions(
 	const { firstName, lastName, email } =
 		state.page.checkoutForm.personalDetails.errors ?? {};
 
+	const otherAmount = getOtherAmountErrors(state);
+	const paymentMethod = state.page.checkoutForm.payment.paymentMethod.errors;
+	const robot_checkbox = state.page.checkoutForm.recaptcha.errors;
+
+	const nonPersonalDetailsErrors = {
+		otherAmount,
+		paymentMethod,
+		...getPaymentMethodErrors(state),
+		robot_checkbox,
+	};
+
 	if (contributionType === 'ONE_OFF') {
 		return {
 			email,
-			...getPaymentMethodErrors(state),
+			...nonPersonalDetailsErrors,
 		};
 	}
 	return {
 		firstName,
 		lastName,
 		email,
```

Comment:
If we're in the US should 'state' be part of the personal details errors? As we collect this with the personal details?

Reply:
Yes, I'm going to tackle this in a future PR as we also need to validate other address details in certain contexts.

Reply:
Ok sounds good